root/types.js

```diff
@@ -421,6 +421,7 @@ declare type PlaceT = {|
   ...CoreEntityRoleT,
   ...TypeRoleT<PlaceTypeT>,
   +entityType: 'place',
+  +address: string | null,
```

Comment:
AFAICT this can't be `null` because it's marked as `NOT NULL` in the schema, and the Moose type in `Entity::Place` isn't `Maybe[str]`.